scripts/build.sh

```diff
@@ -127,32 +127,7 @@ mkdir -p $binary_cache
 # Create the minimal tar bundle
 tar czf $tmp_output_file -C $release_dir contiv-$VERSION
 
```

Comment:
It can be easily retrieved from git's history in that case.  Unused code shouldn't hang around even if it's commented out